Index: FPT.TeamMatching.API/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>ConnectionStrings__DefaultConnection='Server=.; Database=FPT_TeamMatching; Trusted_Connection=true; Integrated Security=True; TrustServerCertificate=True;'\r\nFrontend__Domain='https://localhost:3000'\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/FPT.TeamMatching.API/.env b/FPT.TeamMatching.API/.env
--- a/FPT.TeamMatching.API/.env	(revision 690b8b44335e16470530d856d7f2e5b3eec68d5f)
+++ b/FPT.TeamMatching.API/.env	(date 1739167943279)
@@ -1,2 +1,4 @@
-ConnectionStrings__DefaultConnection='Server=.; Database=FPT_TeamMatching; Trusted_Connection=true; Integrated Security=True; TrustServerCertificate=True;'
-Frontend__Domain='https://localhost:3000'
+ConnectionStrings__DefaultConnection="Server=ep-small-glitter-a1pc2wdu.ap-southeast-1.aws.neon.tech;Database=FPT_TeamMatching;User Id=neondb_owner;Password=npg_Uc9jKvqm2JBF;"
+Frontend__Domain='http://localhost:3000'
+MONGODB_URI='mongodb+srv://loctodaleservice:8841541@cluster0.uefir.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0'
+CLOUDINARY_URL=cloudinary://534873291357266:ZMz5D3aizdu3h7DRtF-tL9pZM5Y@dijvg89ff
\ No newline at end of file
Index: FPT.TeamMatching.API/appsettings.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"TokenSetting\": {\r\n    \"AccessTokenExpiryMinutes\": 30,\r\n    \"RefreshTokenExpiryDays\": 3\r\n  },\r\n  \"AppSettings\": {\r\n    \"Token\": \"4cd3ad315a6f6e1c82e754c7c9d457186335acdf89948342d66b042dac4e8b76c15a8e0d4a3b61daf47b8b9ac1603d80a4c5d8dbc3acc\",\r\n    \"Issuer\": \"FPT.TeamMatching\",\r\n    \"Audience\": \"FPT.TeamMatching\"\r\n  },\r\n  \"ConnectionStrings\": {\r\n    \"DefaultConnection\": \"\"\r\n  },\r\n  \"Frontend\": {\r\n    \"Domain\": \"https://localhost:3000\"\r\n  },\r\n  \"Logging\": {\r\n    \"LogLevel\": {\r\n      \"Default\": \"Information\",\r\n      \"Microsoft.AspNetCore\": \"Warning\"\r\n    }\r\n  },\r\n  \"AllowedHosts\": \"*\"\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/FPT.TeamMatching.API/appsettings.json b/FPT.TeamMatching.API/appsettings.json
--- a/FPT.TeamMatching.API/appsettings.json	(revision 690b8b44335e16470530d856d7f2e5b3eec68d5f)
+++ b/FPT.TeamMatching.API/appsettings.json	(date 1739167947294)
@@ -3,16 +3,11 @@
     "AccessTokenExpiryMinutes": 30,
     "RefreshTokenExpiryDays": 3
   },
-  "AppSettings": {
-    "Token": "4cd3ad315a6f6e1c82e754c7c9d457186335acdf89948342d66b042dac4e8b76c15a8e0d4a3b61daf47b8b9ac1603d80a4c5d8dbc3acc",
-    "Issuer": "FPT.TeamMatching",
-    "Audience": "FPT.TeamMatching"
-  },
   "ConnectionStrings": {
     "DefaultConnection": ""
   },
   "Frontend": {
-    "Domain": "https://localhost:3000"
+    "Domain": "http://localhost:3000"
   },
   "Logging": {
     "LogLevel": {
@@ -20,5 +15,6 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "MONGODB_URI": "mongodb+srv://loctodaleservice:8841541@cluster0.uefir.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0",
   "AllowedHosts": "*"
 }
